src/mp_api/tasks/models.py

```diff
@@ -0,0 +1,109 @@
+from datetime import datetime
+from enum import Enum
+from typing import List
+
+from monty.json import MontyDecoder
+from mp_api.materials.models import Structure
+from pydantic import BaseModel, Field, validator
+
+monty_decoder = MontyDecoder()
+
+
+class TaskType(str, Enum):
+    """
+    The type of calculation
+    """
+
+    EMPTY = ("",)
+    GGA_NSCF_Line = ("GGA NSCF Line",)
+    GGA_NSCF_Uniform = ("GGA NSCF Uniform",)
+    GGA_Static = ("GGA Static",)
+    GGA_Structure_Optimization = ("GGA Structure Optimization",)
+    GGA_U_NSCF_Line = ("GGA+U NSCF Line",)
+    GGA_U_NSCF_Uniform = ("GGA+U NSCF Uniform",)
+    GGA_U_Static = ("GGA+U Static",)
+    GGA_U_Structure_Optimization = "GGA+U Structure Optimization"
```

Comment:
Why is the enum value not a `tuple` here?

Reply:
Sorry there's no reason for this. Also for some reason, it seems to do the same thing whether it's a tuple or not???

Reply:
@munrojm  please add brackets around the last line here. Sorry for the oversight.

Reply:
The member value does not need to be a tuple. It's probably fine here to just be the [descriptive string](https://docs.python.org/3/library/enum.html#using-a-descriptive-string).

Reply:
OK I figured out what happened.  didn't have these as tuples but I have a comma after each line except the last.  Then the automatic linting added the brackets for everyone except the last line.
But these got parsed by fastAPI the same way (for some reason I do not understand) so I never caught the change.